### DISCUSSION TITLE
posts/2022-07-25-archlinux-btrfs-full-disk-encryption-install/

### DISCUSSION THREAD

**Discussion by giscus** [starunity/starunity-blog#11] [Announcements]
# posts/2022-07-25-archlinux-btrfs-full-disk-encryption-install/

ArchLinux Btrfs 磁盘加密安装配置流程

https://blog.starunity.dev/posts/2022-07-25-archlinux-btrfs-full-disk-encryption-install/

**Comment by haowuge** [comment]
nvim /etc/local.gen     <-这个是locale.gen,少输了一个e

**Reply by starunity** [reply]
感谢纠错🎉
